### PR TITLE
Fix: lambda invocation trigger

### PR DESCRIPTION
**Pull request by aputzufulll**

### CODE REVIEW

File: lambda.tf [hunk 1]
```diff
@@ -107,7 +107,7 @@ resource "aws_lambda_invocation" "run_lambda" {
 
   function_name = aws_lambda_function.lambda.function_name
   triggers = {
-    redeployment = filebase64sha256(var.package_type != "Zip" ? var.lambda_filepath : var.image_uri)
+    redeployment = filebase64sha256(var.image_uri == null ? var.lambda_filepath : var.image_uri)
```
Comment (vibou) [inline lambda.tf:110]: please remove the `package_type` variable then which shouldn't be used anywhere else is it?

**Review by vibou — CHANGES_REQUESTED** [review]
A question regarding the `package_type` variable

File: lambda.tf (hunk 1 above)
Comment (aputzufulll) [reply lambda.tf:110]: `package_type` is used in the lambda function :
```
  source_code_hash = var.package_type != "Zip" ? null : filebase64sha256(var.lambda_filepath)
  publish = var.lambda_publish
  s3_bucket     = var.package_type != "Zip" ? null : data.aws_s3_bucket.deployment_bucket[0].id
  s3_key        = var.package_type != "Zip" ? null : var.lambda_bucket_key
  image_uri     = var.package_type != "Zip" ? var.image_uri : null
  package_type  = var.package_type
```

Comment (vibou) [reply lambda.tf:110]: package type can be removed and use a condition based on image_uri

**Review by vibou — APPROVED** [review]

**Pull request merged**